client/homebrew/brewRenderer/safeHTML.js

```diff
@@ -0,0 +1,47 @@
+let doc = null;
+let div = null;
+
+function safeHTML(htmlString) {
+	// If the Document interface doesn't exist, exit
+	if(!document) return null;
+	// If the test document and div don't exist, create them
+	if(!doc) doc = document.implementation.createHTMLDocument('');
+	if(!div) div = doc.createElement('div');
+
+	// Set the test div contents to the evaluation string
+	div.innerHTML = htmlString;
+	// Grab all nodes from the test div
+	const elements = div.querySelectorAll('*');
+
+	// Blacklisted tags
+	const blacklistTags = ['script', 'noscript', 'noembed'];
+	// Tests to remove attributes
+	const blacklistAttrs = [
+		(test)=>{return test.localName.indexOf('on') == 0;},
+		(test)=>{return test.value.replace(/[\u0000-\u0020\u00A0\u1680\u180E\u2000-\u2029\u205f\u3000]/g, '').toLowerCase().trim().indexOf('javascript:') == 0;}
+	];
+
+
+	elements.forEach((element)=>{
+		// Check each element for blacklisted type
+		if(blacklistTags.includes(element?.localName?.toLowerCase())) {
+			element.parentNode.removeChild(element);
```

Comment:
Why not just `element.remove()`, rather than going up to the parent, and back down to the child?

Reply:
`remove` is a newer function that achieves the same result. Some "wisdom of the ancients" shows wider support for the `removeChild` process, which boils down to being supported by a couple of ancient versions of IE.
Ultimately, `remove` is about as widely supported as `removeAttribute` which is used later in the function, so I have no particular issue with changing to use `remove`.